content/docs/use-cases/ci-cd-for-machine-learning.md

```diff
@@ -0,0 +1,98 @@
+# Continuous Integration and Deployment for Machine Learning
+
+Applying DevOps methodologies to machine learning (MLOps) and data management
+(DataOps) is increasingly common. This means resource orchestration
+(provisioning servers for model training), model testing (validating model
+inference), and model deployment to production, as well as monitoring &
+feedback. [DVC](/) can manage data/models and reproducible pipelines, while
+[CML] can assist with orchestration, testing and monitoring.
```

Comment:
I'd leave out the parentheses from this intro paragraph to make it super easy to read. Details can be presented further down if needed or just assume readers who find this use case interesting know what orchestration and model testing involve.